java-practice-methods/.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Unnecessary file (this is the project settings file for IntelliJ IDEA).
Fix it:
- create file .gitignore
- add to the file .idea/